cli.py

```diff
@@ -68,20 +71,50 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
-@click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+@click.option('-i', '--simulation-info', type=str,
+              help="Simulation File info (accept absolute and relative path)")
+@click.option('--run-on-redis', is_flag=True, default=False,
+              help="Start the client using the Redis API")
+def run(base_setup_path, setup_module_name, username, password, domain_name, web_socket,
+        simulation_info, run_on_redis, **kwargs):
```

Comment:
Minor, but in order to show the reader that this is a file_path_str, how abour renaming this inputvariable (and consequently the clit parameter) to somwthing like:
`simulation_info_path` or even `simulation_info_file_path?`

Reply:
It could be the filename as well as path.

Reply:
Even the filename can be considered a 'path' (relative path to the current directory). I would also prefer if this variable was named simulation_info_path (I actually care more about its appearance as a CLI argument rather than in the function argument, I think that `--simulation-info-path` is much more clear than `--simulation-info` for the user of the CLI).

Moving forward to this direction, this is really not info, right ? As far as I understand this is a configuration file (it is used in order to preconfigure the connection to a simulation), therefore it looks to my eyes that a name `--simulation-config-path` might be more appropriate, assuming that I understood the intent of this file correctly.